cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/scopes/Scope.kt

```diff
@@ -210,6 +213,33 @@ sealed class Scope(
                 }
         }
 
+        // If the symbol was still not resolved, and we are performing an unqualified resolution, we
+        // search in the
+        // languages builtin scope for the symbol
+        if (list.isNullOrEmpty() && !qualifiedLookup) {
+            list = mutableListOf()
+            // If the language has builtins we can search there for the symbol
+            (languageOnly as? HasBuiltins)?.let {
+                val builtinNamespace = it.builtinsNamespace
+                // Retrieve the builtins scope from the builtins namespace
+                (ctx ?: this.astNode?.ctx)?.scopeManager?.lookupScope(builtinNamespace)?.let {
+                    builtinScope ->
+                    // Obviously we don't want to search in the builtins scope if we already failed
+                    // finding the symbol in the builtins scope
+                    if (builtinScope != this) {
+                        list.addAll(
```

Comment:
I think you can just set use `list = ... lookupSymbol`, then you don't need to initialize the list in line 220

Reply:
Done